acceptance_tests/features/steps/refusal.py

```diff
@@ -32,3 +33,8 @@ def send_refusal_msg(context):
             message=message,
             content_type='application/json',
             routing_key=Config.RABBITMQ_REFUSAL_QUEUE)
+
+
+@step("the events logged for the refused case are {expected_event_list}")
+def check_survey_launch_event_logging(context, expected_event_list):
```

Comment:
Bad function name. This step also seems to be duplicated, I would suggest writing it once in an events step file